client/src/app/components/form/form.component.html

```diff
@@ -0,0 +1,63 @@
+<h3>Zgłoś nieobecność</h3>
+<form [formGroup]="form" novalidate (submit)='onSubmit()'>
+    <div class="form-group">
+        <label class="center-block">Name:
+            <input id="name" class="form-control" formControlName="name" required>
```

Comment:
Validation is done on the component side; the `required` attribute does nothing here. Remove from all inputs.